packages/tao/src/commands/migrate.ts

```diff
@@ -524,9 +529,14 @@ function updatePackageJson(
       json.devDependencies[p] = updatedPackages[p].version;
     } else if (json.dependencies && json.dependencies[p]) {
       json.dependencies[p] = updatedPackages[p].version;
-    } else if (updatedPackages[p].alwaysAddToPackageJson) {
-      if (!json.dependencies) json.dependencies = {};
-      json.dependencies[p] = updatedPackages[p].version;
+    } else if (updatedPackages[p].addToPackageJson) {
+      if (updatedPackages[p].addToPackageJson === 'dependencies') {
```

Comment:
What about the case for `addToPackageJson === true`? This is used in one of the nest migrations.

The previous semantics is `true` means adding to `dependencies`, so we need to keep that logic intact for old migrations.

Maybe we should add `isDevDep` flag instead? This it is just an addition to the existing logic.

Reply:
`addToPackageJson` is the new property though, nothing sets it yet. and `true` isn't a valid value for that new property - it's type is `'dependencies' | 'devDependencies' | false`.

---

originally this PR added a new option `devDependency: boolean` property, but after discussing it with @FrozenPandaz , we decided to add this `addToPackageJson` property instead, that replaces the existing `alwaysAddToPackageJson`